src/modules/newLike.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-useless-concat */
```

Comment:
- [ ] [OPTIONAL] Please remove all the disabled linters and fix the linters for better practices and clean code, I will not require you to change them because you fixed the specifically pointed linters by the previous reviewer, this is the last review you have, and the deadline is near however I highly recommend fixing this linters instead of disabling them